networkx/algorithms/tests/test_matching.py

```diff
@@ -63,9 +63,6 @@ def test_path(self):
         assert edges_equal(
             nx.min_weight_matching(G), matching_dict_to_set({1: 2, 3: 4})
         )
-        assert edges_equal(
-            nx.min_weight_matching(G, 1), matching_dict_to_set({1: 2, 3: 4})
-        )
```

Comment:
Rather than deleting this, I think the `1` should be changed to `weight=None` instead, which I suspect was the original intent of the test (changing `1`->`weight=None` should be done for the max_weight_matching test a few lines above as well!).

Reply:
I wouldn't want to codify `True` in the test suite though - the docstring makes it clear that `weight` is intended to be a string indicating the attribute name, therefore the test suite should reflect this. The fact that it "works" with other values is just luck, not intended behavior.